exec/context_test.go

```diff
@@ -70,3 +71,19 @@ func Test_Context_Populate(t *testing.T) {
 	_, moreItems := <-testContextChannel
 	a.False(moreItems)
 }
+
+func Test_RandStringBytes(t *testing.T) {
```

Comment:
Nice tests! 
This test covers now more than one usecase, maybe you want to split in different tests or use [TableDrivenTests](https://github.com/golang/go/wiki/TableDrivenTests)?